skimage/transform/_warps.py

```diff
@@ -371,7 +371,7 @@ def rotate(image, angle, resize=False, center=None, order=1, mode='constant',
         maxr = corners[:, 1].max()
         out_rows = maxr - minr + 1
         out_cols = maxc - minc + 1
-        output_shape = np.ceil((out_rows, out_cols))
+        output_shape = np.round((out_rows, out_cols))
```

Comment:
Not sure about this. In Python 3 this rounds to the nearest even value, which means that the function might still produce a differently shaped output if some dimensions of the input image are odd. Should this line use `floor` instead?

P.S. To the best of my knowledge, `np.round` is deprecated, and `np.around` is recommended instead.

Reply:
The even/odd behavior is not an issue, since it only takes effect at the 0.5 boundary, otherwise the behavior is expected. np.round/np.around is the correct choice here.